Yesterday, we looked at a scRNA-seq dataset of <i style="color:#EB1960">Danio rerio</i> cells [@zebrafish-data].  We spent a lot of time understanding how the dataset was created, and ended with a bit of a mystery: <i style="color:#C0CF96">why did the cell counts not match up between our datasets?</i>  I don't have an answer to that mystery, unfortunately - but in my experience this mismatch happens in a lot of papers.  We might as well continue with the analysis, rather than getting hung up on a minor anomaly.

I realized since yesterday that there was another gene counts file, (the filtered tpms file).  I downloaded this and the "experiment metadata" files; I suspect these may be useful as I know that the TPM values were pre-quality-control:

> For the <b style="color:#EB1960">Smart-seq2 protocol</b> transcript per million (TPM) values reported by Salmon were used for the quality control (QC). Wells with fewer than 900 expressed genes (TPM > 1) or having more than either 60% of ERCC or 45% of mitochondrial content were annotated as <strong style="color:#C0CF96">poor quality cells</strong>. As a result, 322 cells failed QC and 542 single cells were selected for the further study.
>
> -- <cite><b style="color:#EB1960">Quality Control of Single-Cell Data; Materials and Methods Section;</b> @zebrafish-data </cite>

<details>
    <summary style="color:#C0CF96">
        <b>Can I mix R and Python in the same notebook?</b>
    </summary>
    Yes!  I sometimes use <b style="color:#EB1960">SOS Kernel</b> which allows me to swap between kernels at ease.  You can change kernels (R vs Python) using <span style="color:#757575">%use</span> magic commands; I've done this before but the syntax highlighting gets messed up often, so I've chosen not to. do that here.
</details>

In [ ]:
raw.counts <- Matrix::readMM('./localdata/E-MTAB-7117.expression_tpm.mtx')
dim(raw.counts)

[1] 21797   966

In [ ]:
%%html
<style>
div.input {
    display:none;
}
</style>

ERROR: Error in parse(text = x, srcfile = src): <text>:1:1: unexpected SPECIAL
1: %%
    ^


#### <span style="color:#C0CF96">Does the metadata file help us resolve the cell count mystery?</span>

What does this extra metadata file tell us?  Let's see:

In [ ]:
meta.data <- read.table("./localdata/E-MTAB-7117.sdrf.txt", sep='\t', header = TRUE)
head(meta.data)
dim(meta.data)
length(which(meta.data['Characteristics.single.cell.quality.'] != "not OK"))

Source.Name  Comment.ENA_SAMPLE. Comment.BioSD_SAMPLE.
1 CD4_gill_A1  ERS2634491          SAMEA4814592         
2 CD4_gill_A1  ERS2634491          SAMEA4814592         
3 CD4_gill_A10 ERS2634691          SAMEA4814793         
4 CD4_gill_A10 ERS2634691          SAMEA4814793         
5 CD4_gill_A11 ERS2635221          SAMEA4815323         
6 CD4_gill_A11 ERS2635221          SAMEA4815323         
  Characteristics.organism. Characteristics.strain. Characteristics.age.
1 Danio rerio               AB                      6                   
2 Danio rerio               AB                      6                   
3 Danio rerio               AB                      6                   
4 Danio rerio               AB                      6                   
5 Danio rerio               AB                      6                   
6 Danio rerio               AB                      6                   
  Unit.time.unit. Characteristics.developmental.stage. Characteristics.sex.
1 month           adult                                male                
2 month           adult                                male                
3 month           adult                                male                
4 month           adult                                male                
5 month           adult                                male                
6 month           adult                                male                
  Characteristics.genotype. ⋯ Comment.ENA_EXPERIMENT.
1 Tg(cd4-1:mCherry)         ⋯ ERX2737040             
2 Tg(cd4-1:mCherry)         ⋯ ERX2737040             
3 Tg(cd4-1:mCherry)         ⋯ ERX2737240             
4 Tg(cd4-1:mCherry)         ⋯ ERX2737240             
5 Tg(cd4-1:mCherry)         ⋯ ERX2737770             
6 Tg(cd4-1:mCherry)         ⋯ ERX2737770             
  Scan.Name                                  
1 SLX-10875.N701_S513.C9FTNANXX.s_5.r_1.fq.gz
2 SLX-10875.N701_S513.C9FTNANXX.s_5.r_2.fq.gz
3 SLX-10875.N712_S513.C9FTNANXX.s_5.r_1.fq.gz
4 SLX-10875.N712_S513.C9FTNANXX.s_5.r_2.fq.gz
5 SLX-10875.N714_S513.C9FTNANXX.s_5.r_1.fq.gz
6 SLX-10875.N714_S513.C9FTNANXX.s_5.r_2.fq.gz
  Comment.SUBMITTED_FILE_NAME.                Comment.ENA_RUN.
1 SLX-10875.N701_S513.C9FTNANXX.s_5.r_1.fq.gz ERR2723271      
2 SLX-10875.N701_S513.C9FTNANXX.s_5.r_2.fq.gz ERR2723271      
3 SLX-10875.N712_S513.C9FTNANXX.s_5.r_1.fq.gz ERR2723471      
4 SLX-10875.N712_S513.C9FTNANXX.s_5.r_2.fq.gz ERR2723471      
5 SLX-10875.N714_S513.C9FTNANXX.s_5.r_1.fq.gz ERR2724001      
6 SLX-10875.N714_S513.C9FTNANXX.s_5.r_2.fq.gz ERR2724001      
  Comment.FASTQ_URI.                                                            
1 ftp://ftp.sra.ebi.ac.uk/vol1/fastq/ERR272/001/ERR2723271/ERR2723271_1.fastq.gz
2 ftp://ftp.sra.ebi.ac.uk/vol1/fastq/ERR272/001/ERR2723271/ERR2723271_2.fastq.gz
3 ftp://ftp.sra.ebi.ac.uk/vol1/fastq/ERR272/001/ERR2723471/ERR2723471_1.fastq.gz
4 ftp://ftp.sra.ebi.ac.uk/vol1/fastq/ERR272/001/ERR2723471/ERR2723471_2.fastq.gz
5 ftp://ftp.sra.ebi.ac.uk/vol1/fastq/ERR272/001/ERR2724001/ERR2724001_1.fastq.gz
6 ftp://ftp.sra.ebi.ac.uk/vol1/fastq/ERR272/001/ERR2724001/ERR2724001_2.fastq.gz
  Comment.SPOT_LENGTH. Comment.READ_INDEX_1_BASE_COORD. Factor.Value.genotype.
1 250                  126                              Tg(cd4-1:mCherry)     
2 250                  126                              Tg(cd4-1:mCherry)     
3 250                  126                              Tg(cd4-1:mCherry)     
4 250                  126                              Tg(cd4-1:mCherry)     
5 250                  126                              Tg(cd4-1:mCherry)     
6 250                  126                              Tg(cd4-1:mCherry)     
  Factor.Value.organism.part. Factor.Value.single.cell.identifier.
1 gill                        CD4_gill_A1                         
2 gill                        CD4_gill_A1                         
3 gill                        CD4_gill_A10                        
4 gill                        C

[1] 2112   55

[1] 1952

At first glance, <b style="color:#C0CF96">no</b> - it does not.  The metadata has 2112 lines, and removing poor quality cells leaves us with 1952.

But, it does seem like some cells have multiple names, so let's filter by unique values.

In [ ]:
dim(unique(meta.data['Source.Name']))
length(
    unique(
        meta.data[
            which(meta.data['Characteristics.single.cell.quality.'] != "not OK"),
            "Source.Name"
        ]
    )
)

[1] 1056    1

[1] 976

Gee whiz, that 976 number returns...  We can re-use the list of 10 extra cells calculated yesterday to investigate them:

In [ ]:
discrepancies <- c(
    "ERR2723217", "ERR2723218", "ERR2723236", "ERR2723237", "ERR2723551",
    "ERR2723578", "ERR2723789", "ERR2723794", "ERR2723974", "ERR2723990"
)
extra.cells <- meta.data[
    which(sapply(meta.data["Comment.ENA_RUN."], `%in%`, discrepancies)),
]

In [ ]:
#| output: false
# Output hidden to reduce spam
extra.cells[1:10]
extra.cells[11:20]
extra.cells[21:30]
extra.cells[31:40]
extra.cells[41:55]

Source.Name           Comment.ENA_SAMPLE. Comment.BioSD_SAMPLE.
247  CD4_gut_D4            ERS2634771          SAMEA4814873         
248  CD4_gut_D4            ERS2634771          SAMEA4814873         
375  CD4_kidney_C5         ERS2634798          SAMEA4814900         
376  CD4_kidney_C5         ERS2634798          SAMEA4814900         
1053 LCK_gut_G8            ERS2634437          SAMEA4814538         
1054 LCK_gut_G8            ERS2634437          SAMEA4814538         
1055 LCK_gut_G9            ERS2634438          SAMEA4814539         
1056 LCK_gut_G9            ERS2634438          SAMEA4814539         
1305 LCK_thymus_B2         ERS2635210          SAMEA4815312         
1306 LCK_thymus_B2         ERS2635210          SAMEA4815312         
1333 LCK_thymus_C4         ERS2634456          SAMEA4814557         
1334 LCK_thymus_C4         ERS2634456          SAMEA4814557         
1335 LCK_thymus_C5         ERS2634457          SAMEA4814558         
1336 LCK_thymus_C5         ERS2634457          SAMEA4814558         
1545 mhc2dab_kidney_pl2_A2 ERS2635009          SAMEA4815111         
1546 mhc2dab_kidney_pl2_A2 ERS2635009          SAMEA4815111         
1609 mhc2dab_kidney_pl2_D1 ERS2635014          SAMEA4815116         
1610 mhc2dab_kidney_pl2_D1 ERS2635014          SAMEA4815116         
2099 Rag1_kidney_H3        ERS2635194          SAMEA4815296         
2100 Rag1_kidney_H3        ERS2635194          SAMEA4815296         
     Characteristics.organism. Characteristics.strain. Characteristics.age.
247  Danio rerio               AB                       6                  
248  Danio rerio               AB                       6                  
375  Danio rerio               AB                       6                  
376  Danio rerio               AB                       6                  
1053 Danio rerio               AB                       6                  
1054 Danio rerio               AB                       6                  
1055 Danio rerio               AB                       6                  
1056 Danio rerio               AB                       6                  
1305 Danio rerio               AB                       6                  
1306 Danio rerio               AB                       6                  
1333 Danio rerio               AB                       6                  
1334 Danio rerio               AB                       6                  
1335 Danio rerio               AB                       6                  
1336 Danio rerio               AB                       6                  
1545 Danio rerio               AB                      10                  
1546 Danio rerio               AB                      10                  
1609 Danio rerio               AB                      10                  
1610 Danio rerio               AB                      10                  
2099 Danio rerio               AB                       4                  
2100 Danio rerio               AB                       4                  
     Unit.time.unit. Characteristics.developmental.stage. Characteristics.sex.
247  month           adult                                male                
248  month           adult                                male                
375  month           adult                                male                
376  month           adult                                male                
1053 month           adult                                male                
1054 month           adult                                male                
1055 month           adult                                male                
1056 month           adult                                male                
1305 month           adult                                male                
1306 month           adult                                male                
1333 month           adult                                male                
1334 month  

Characteristics.organism.part. Characteristics.cell.type.
247  gut                            blood cell                
248  gut                            blood cell                
375  kidney                         blood cell                
376  kidney                         blood cell                
1053 gut                            blood cell                
1054 gut                            blood cell                
1055 gut                            blood cell                
1056 gut                            blood cell                
1305 thymus                         blood cell                
1306 thymus                         blood cell                
1333 thymus                         blood cell                
1334 thymus                         blood cell                
1335 thymus                         blood cell                
1336 thymus                         blood cell                
1545 kidney                         blood cell                
1546 kidney                         blood cell                
1609 kidney                         blood cell                
1610 kidney                         blood cell                
2099 kidney                         blood cell                
2100 kidney                         blood cell                
     Characteristics.phenotype.  Characteristics.individual.
247  mCherry positive cell       2                          
248  mCherry positive cell       2                          
375  mCherry positive cell       2                          
376  mCherry positive cell       2                          
1053 EGFP positive cell          1                          
1054 EGFP positive cell          1                          
1055 EGFP positive cell          1                          
1056 EGFP positive cell          1                          
1305 EGFP positive cell          1                          
1306 EGFP positive cell          1                          
1333 EGFP positive cell          1                          
1334 EGFP positive cell          1                          
1335 EGFP positive cell          1                          
1336 EGFP positive cell          1                          
1545 GFP and dsRed positive cell 3                          
1546 GFP and dsRed positive cell 3                          
1609 GFP and dsRed positive cell 3                          
1610 GFP and dsRed positive cell 3                          
2099 EGFP positive cell          4                          
2100 EGFP positive cell          4                          
     Characteristics.well.information. Characteristics.single.cell.quality.
247  single cell                       OK                                  
248  single cell                       OK                                  
375  single cell                       OK                                  
376  single cell                       OK                                  
1053 single cell                       OK                                  
1054 single cell                       OK                                  
1055 single cell                       OK                                  
1056 single cell                       OK                                  
1305 single cell                       OK                                  
1306 single cell                       OK                                  
1333 single cell                       OK                                  
1334 single cell                       OK                                  
1335 single cell                       OK                                  
1336 single cell                       OK                                  
1545 single cell                       OK                                  
1546 single cell                       OK                                  
1609 single cell                       OK                                  
1610 single cell                 

Protocol.REF Performer          Protocol.REF.1 Performer.1       
247  P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
248  P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
375  P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
376  P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1053 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1054 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1055 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1056 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1305 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1306 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1333 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1334 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1335 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1336 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1545 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1546 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1609 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
1610 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
2099 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
2100 P-MTAB-77662 Paulina Strzelecka P-MTAB-77663   Paulina Strzelecka
     Protocol.REF.2 Performer.2        Extract.Name          Material.Type.1
247  P-MTAB-77664   Paulina Strzelecka CD4_gut_D4            RNA            
248  P-MTAB-77664   Paulina Strzelecka CD4_gut_D4            RNA            
375  P-MTAB-77664   Paulina Strzelecka CD4_kidney_C5         RNA            
376  P-MTAB-77664   Paulina Strzelecka CD4_kidney_C5         RNA            
1053 P-MTAB-77664   Paulina Strzelecka LCK_gut_G8            RNA            
1054 P-MTAB-77664   Paulina Strzelecka LCK_gut_G8            RNA            
1055 P-MTAB-77664   Paulina Strzelecka LCK_gut_G9            RNA            
1056 P-MTAB-77664   Paulina Strzelecka LCK_gut_G9            RNA            
1305 P-MTAB-77664   Paulina Strzelecka LCK_thymus_B2         RNA            
1306 P-MTAB-77664   Paulina Strzelecka LCK_thymus_B2         RNA            
1333 P-MTAB-77664   Paulina Strzelecka LCK_thymus_C4         RNA            
1334 P-MTAB-77664   Paulina Strzelecka LCK_thymus_C4         RNA            
1335 P-MTAB-77664   Paulina Strzelecka LCK_thymus_C5         RNA            
1336 P-MTAB-77664   Paulina Strzelecka LCK_thymus_C5         RNA            
1545 P-MTAB-77664   Paulina Strzelecka mhc2dab_kidney_pl2_A2 RNA            
1546 P-MTAB-77664   Paulina Strzelecka mhc2dab_kidney_pl2_A2 RNA            
1609 P-MTAB-77664   Paulina Strzelecka mhc2dab_kidney_pl2_D1 RNA            
1610 P-MTAB-77664   Paulina Strzelecka mhc2dab_kidney_pl2_D1 RNA            
2099 P-MTAB-77664   Paulina Strzelecka Rag1_kidney_H3        RNA            
2100 P-MTAB-77664   Paulina Strzelecka Rag1_kidney_H3        RNA            
     Comment.single.cell.isolation. Comment.library.construction.
247  FACS                           smart-seq2                   
248  FACS                           smart-seq2                   
375  FACS                           smart-seq2                   
376  FACS                           smart-seq2                   
1053 FACS                           smart-seq2                   
1054 FACS                           smart-seq2                   
1055 FACS                           smart-seq2                   
1056 FACS                           smart-seq2                   
1305 FACS                           smart-seq2                   
1306 FACS                           smart-seq2                   
1333 FACS                           smart-seq2                   
1334 FACS                           smart-seq2                   
1335 FACS                           sma

Comment.input.molecule. Comment.primer. Comment.end.bias.
247  polyA RNA               oligo-dT        none             
248  polyA RNA               oligo-dT        none             
375  polyA RNA               oligo-dT        none             
376  polyA RNA               oligo-dT        none             
1053 polyA RNA               oligo-dT        none             
1054 polyA RNA               oligo-dT        none             
1055 polyA RNA               oligo-dT        none             
1056 polyA RNA               oligo-dT        none             
1305 polyA RNA               oligo-dT        none             
1306 polyA RNA               oligo-dT        none             
1333 polyA RNA               oligo-dT        none             
1334 polyA RNA               oligo-dT        none             
1335 polyA RNA               oligo-dT        none             
1336 polyA RNA               oligo-dT        none             
1545 polyA RNA               oligo-dT        none             
1546 polyA RNA               oligo-dT        none             
1609 polyA RNA               oligo-dT        none             
1610 polyA RNA               oligo-dT        none             
2099 polyA RNA               oligo-dT        none             
2100 polyA RNA               oligo-dT        none             
     Comment.LIBRARY_LAYOUT. Comment.LIBRARY_SELECTION.
247  PAIRED                  Oligo-dT                  
248  PAIRED                  Oligo-dT                  
375  PAIRED                  Oligo-dT                  
376  PAIRED                  Oligo-dT                  
1053 PAIRED                  Oligo-dT                  
1054 PAIRED                  Oligo-dT                  
1055 PAIRED                  Oligo-dT                  
1056 PAIRED                  Oligo-dT                  
1305 PAIRED                  Oligo-dT                  
1306 PAIRED                  Oligo-dT                  
1333 PAIRED                  Oligo-dT                  
1334 PAIRED                  Oligo-dT                  
1335 PAIRED                  Oligo-dT                  
1336 PAIRED                  Oligo-dT                  
1545 PAIRED                  Oligo-dT                  
1546 PAIRED                  Oligo-dT                  
1609 PAIRED                  Oligo-dT                  
1610 PAIRED                  Oligo-dT                  
2099 PAIRED                  Oligo-dT                  
2100 PAIRED                  Oligo-dT                  
     Comment.LIBRARY_SOURCE.    Comment.LIBRARY_STRAND.
247  TRANSCRIPTOMIC SINGLE CELL not applicable         
248  TRANSCRIPTOMIC SINGLE CELL not applicable         
375  TRANSCRIPTOMIC SINGLE CELL not applicable         
376  TRANSCRIPTOMIC SINGLE CELL not applicable         
1053 TRANSCRIPTOMIC SINGLE CELL not applicable         
1054 TRANSCRIPTOMIC SINGLE CELL not applicable         
1055 TRANSCRIPTOMIC SINGLE CELL not applicable         
1056 TRANSCRIPTOMIC SINGLE CELL not applicable         
1305 TRANSCRIPTOMIC SINGLE CELL not applicable         
1306 TRANSCRIPTOMIC SINGLE CELL not applicable         
1333 TRANSCRIPTOMIC SINGLE CELL not applicable         
1334 TRANSCRIPTOMIC SINGLE CELL not applicable         
1335 TRANSCRIPTOMIC SINGLE CELL not applicable         
1336 TRANSCRIPTOMIC SINGLE CELL not applicable         
1545 TRANSCRIPTOMIC SINGLE CELL not applicable         
1546 TRANSCRIPTOMIC SINGLE CELL not applicable         
1609 TRANSCRIPTOMIC SINGLE CELL not applicable         
1610 TRANSCRIPTOMIC SINGLE CELL not applicable         
2099 TRANSCRIPTOMIC SINGLE CELL not applicable         
2100 TRANSCRIPTOMIC SINGLE CELL not applicable         
     Comment.LIBRARY_STRATEGY. Comment.NOMINAL_LENGTH. Comment.NOMINAL_SDEV.
247  RNA-Seq                   400                     20                   
248  RNA-Seq                   400                     20                   
375  RNA-Seq                   400                     20                   
376  RNA-Seq          

Comment.ORIENTATION. Protocol.REF.3
247  5-3-5-3              P-MTAB-77665  
248  5-3-5-3              P-MTAB-77665  
375  5-3-5-3              P-MTAB-77665  
376  5-3-5-3              P-MTAB-77665  
1053 5-3-5-3              P-MTAB-77665  
1054 5-3-5-3              P-MTAB-77665  
1055 5-3-5-3              P-MTAB-77665  
1056 5-3-5-3              P-MTAB-77665  
1305 5-3-5-3              P-MTAB-77665  
1306 5-3-5-3              P-MTAB-77665  
1333 5-3-5-3              P-MTAB-77665  
1334 5-3-5-3              P-MTAB-77665  
1335 5-3-5-3              P-MTAB-77665  
1336 5-3-5-3              P-MTAB-77665  
1545 5-3-5-3              P-MTAB-77665  
1546 5-3-5-3              P-MTAB-77665  
1609 5-3-5-3              P-MTAB-77665  
1610 5-3-5-3              P-MTAB-77665  
2099 5-3-5-3              P-MTAB-77665  
2100 5-3-5-3              P-MTAB-77665  
     Performer.3                                Assay.Name           
247  Cambridge Institute Genomics Core Facility CD4_gut_D4           
248  Cambridge Institute Genomics Core Facility CD4_gut_D4           
375  Cambridge Institute Genomics Core Facility CD4_kidney_C5        
376  Cambridge Institute Genomics Core Facility CD4_kidney_C5        
1053 Cambridge Institute Genomics Core Facility LCK_gut_G8           
1054 Cambridge Institute Genomics Core Facility LCK_gut_G8           
1055 Cambridge Institute Genomics Core Facility LCK_gut_G9           
1056 Cambridge Institute Genomics Core Facility LCK_gut_G9           
1305 Cambridge Institute Genomics Core Facility LCK_thymus_B2        
1306 Cambridge Institute Genomics Core Facility LCK_thymus_B2        
1333 Cambridge Institute Genomics Core Facility LCK_thymus_C4        
1334 Cambridge Institute Genomics Core Facility LCK_thymus_C4        
1335 Cambridge Institute Genomics Core Facility LCK_thymus_C5        
1336 Cambridge Institute Genomics Core Facility LCK_thymus_C5        
1545 Cambridge Institute Genomics Core Facility mhc2dab_kidney_pl2_A2
1546 Cambridge Institute Genomics Core Facility mhc2dab_kidney_pl2_A2
1609 Cambridge Institute Genomics Core Facility mhc2dab_kidney_pl2_D1
1610 Cambridge Institute Genomics Core Facility mhc2dab_kidney_pl2_D1
2099 Cambridge Institute Genomics Core Facility Rag1_kidney_H3       
2100 Cambridge Institute Genomics Core Facility Rag1_kidney_H3       
     Technology.Type  Comment.ENA_EXPERIMENT.
247  sequencing assay ERX2737320             
248  sequencing assay ERX2737320             
375  sequencing assay ERX2737347             
376  sequencing assay ERX2737347             
1053 sequencing assay ERX2736986             
1054 sequencing assay ERX2736986             
1055 sequencing assay ERX2736987             
1056 sequencing assay ERX2736987             
1305 sequencing assay ERX2737759             
1306 sequencing assay ERX2737759             
1333 sequencing assay ERX2737005             
1334 sequencing assay ERX2737005             
1335 sequencing assay ERX2737006             
1336 sequencing assay ERX2737006             
1545 sequencing assay ERX2737558             
1546 sequencing assay ERX2737558             
1609 sequencing assay ERX2737563             
1610 sequencing assay ERX2737563             
2099 sequencing assay ERX2737743             
2100 sequencing assay ERX2737743             
     Scan.Name                                  
247  SLX-12114.i720_i506.HKKG2BBXX.s_1.r_1.fq.gz
248  SLX-12114.i720_i506.HKKG2BBXX.s_1.r_2.fq.gz
375  SLX-10874.N705_S505.C9FTNANXX.s_4.r_1.fq.gz
376  SLX-10874.N705_S505.C9FTNANXX.s_4.r_2.fq.gz
1053 SLX-12119.i710_i521.HKCTNBBXX.s_1.r_1.fq.gz
1054 SLX-12119.i710_i521.HKCTNBBXX.s_1.r_2.fq.gz
1055 SLX-12119.i711_i521.HKCTNBBXX.s_1.r_1.fq.gz
1056 SLX-12119.i711_i521.HKCTNBBXX.s_1.r_2.fq.gz
1305 SLX-12119.i702_i503.HKCTNBBXX.s_1.r_1.fq.gz
1306 SLX-12119.i702_i503.HKCTNBBXX.s_1.r_2.fq.gz
1333 SLX-12119.i704_i505.HKCTNBBXX.s_1.r_1.fq.gz
1334 SLX-12119.i704_i505.HKCTNBBXX.s_1.r_2.fq.gz
1335 SLX-12119.i705_i505.HKCTNBBXX.s_1.r_1.fq.gz
1336 SLX-12119.i705_i5

But as before there doesn't really seem to be any outstanding features.  So the mystery continues!

# Quality Control

### References

::: {#refs}
:::

<script src="https://giscus.app/client.js"
        data-repo="baileyandrew/blog"
        data-repo-id="R_kgDOInJwKg"
        data-category="Announcements"
        data-category-id="DIC_kwDOInJwKs4CTGOQ"
        data-mapping="title"
        data-strict="0"
        data-reactions-enabled="1"
        data-emit-metadata="0"
        data-input-position="top"
        data-theme="dark_protanopia"
        data-lang="en"
        data-loading="lazy"
        crossorigin="anonymous"
        async>
</script>